tests/jax/components/training/losses_test.py

```diff
@@ -0,0 +1,191 @@
+from types import SimpleNamespace
+from typing import Callable, Tuple
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from mava.components.jax.training.losses import (
+    MAPGTrustRegionClippingLossConfig,
+    MAPGWithTrustRegionClippingLoss,
+)
+from mava.systems.jax.trainer import Trainer
+
+
+class Mock_net:
+    """Creates a mock network for loss function"""
+
+    def apply(
+        parameters: jnp.array, observation: jnp.array
+    ) -> Tuple[jnp.ndarray, jnp.ndarray]:
+        """Mock function to apply the network to training data"""
+        return jnp.array(observation), jnp.array(observation)
```

Comment:
They're already jnp arrays

Reply:
There was an error for `observation = SimpleNamespace(observation= jnp.array([0.5, 0.5, 0.7, 0.2]))`  where I didn't set the observation to be a jnp array. It's been fixed and the extra `jnp.array(...)`s have been removed.